lib/shared/src/chat/prompts/index.ts

```diff
@@ -57,6 +57,7 @@ export interface MyPromptsJSON {
 
 // The blueprint of a Cody Command
 export interface CodyPrompt {
+    request_id?: string
```

Comment:
Would it be possible to use camel-case everywhere in the PR?

```suggestion
    requestID?: string
```